match/v1/partners.proto

```diff
@@ -21,6 +25,10 @@ message HeadlessPartner {
   string public_key = 1;
   // Required. Token used to authenticate the external partner.
   string token = 2;
+  // Required. Organization Name (display name of the tenant)
+  string organization_name = 3;
+  // Required. Node_id (display name of the tenant)
```

Comment:
why are they both display name of the tenant?

Reply:
Yeah I'll just change that